ykcapi/src/lib.rs

```diff
@@ -95,29 +100,66 @@ impl Registers {
     }
 }
 
+#[derive(Debug)]
+#[repr(C)]
+struct AOTVar {
+    bbidx: u32,
+    instridx: u32,
+    fname: *const i8,
```

Comment:
Should this be `*const os::raw::c_char`?

Reply:
@ptersilie Is it because there is no `char` type in LLVM?

Reply:
It's a `*i8` in LLVM. Is there a difference?

Reply:
Ah, if it's definitely an `i8` in LLVM, we can do the same here.

[IIRC `char` in C99 is a synonym for `i8`, which is why I raised the query, but I might be wrong on that detail!]

Reply:
Only a difference of intention, I'd say.

Reply:
"a difference of intention" That sounds deep but I don't know what it means!

Reply:
It makes clear that our intention is to store characters, even if there is no difference whatsoever to the compiler.